xonsh/base_shell.py

```diff
@@ -506,15 +506,15 @@ def prompt(self):
         return p
 
     def format_color(self, string, hide=False, force_string=False, **kwargs):
-        """Formats the colors in a string. This base implmentation colors based
-        on ANSI color codes
+        """Formats the colors in a string. This base implementation's colors are
```

Comment:
Since this method gets inherited by other classes which seem to change the colour code standard that they receive as input, it does seem important to differentiate here. I think the word "this" could be changed to "the" though.